tests/test_cdp.py

```diff
@@ -32,7 +32,7 @@ async def test_cdp_sendCommand_resultReturned(websocket):
             }
         })
 
-    assert {"targetInfos": ANY} == command_result
+    assert {"result": ANY} == command_result
```

Comment:
Why is this drastically different now?

Should it now become:

`{"result": {"targetInfos": ANY}}`

?